CONTRIBUTING.md

```diff
@@ -66,35 +66,33 @@ Or to test a single feature:
 
     php behat.phar features/core.feature
 
-More info can be found from `php behat.phar --help`.
+More information can be found by running `php behat.phar --help`.
 
-Versioning ...
+Versioning 
 --------------
 
 ### Versions 
 
-In keeping with the standards of semantic versioning, backward-incompatible fixes will be targeted to "Major" versions. "Minor" versions will be reserved for significant feature/bug releases needed between major versions. "Patch" releases will be reserved only for critical security issues and other bugs critical to stabilizing the release. 
+In keeping with the standards of semantic versioning, backward-incompatible fixes are targeted to "Major" versions. "Minor" versions are reserved for significant feature/bug releases needed between major versions. "Patch" releases are reserved only for critical security issues and other bugs critical to stabilizing the release. 
 
-After a new major version is released, previous major versions will be actively supported for 1 year. 
+After a new major version is released, previous major versions are actively supported for 1 year. 
 
-What qualifies as a backward incompatible change?
+#### What qualifies as a backward incompatible change?
 
-Our initial commitment will be to command compatibility and parameter compatibility. However, since on the command line STDOUT and STDERR are essentially APIs we will make a best effort to keep machine-readable output compatibility, meaning if your code interfaces with Terminus via --json or --bash formats we will try our best to ensure these are stable and compatible between minor release. However, changes to the STDOUT, like success and fail messages, will not be consider incompatible. 
+Our initial commitment is to command compatibility and parameter compatibility. However, since on the command line STDOUT and STDERR are essentially APIs, we will make a best effort to keep machine-readable output compatibility. If your code interfaces with Terminus via --json or --bash formats, we will try our best to ensure these are stable and compatible between minor release. However, changes to the STDOUT, like success and fail messages, are not considered incompatible. 
 
-### Version branches 
+### Version Branches 
 
-If you are using terminus in a production environment you should be deploying the executable for the latest release. ( github.com/pantheon-systems/cli/releases )
+If you are using Terminus in a production environment, you should be deploying the executable for the latest release. ( github.com/pantheon-systems/cli/releases )
```

Comment:
Is the URL supposed to be a link for "the latest release"?
The URL needs an `https://`.